src/main/java/net/ccbluex/liquidbounce/features/module/modules/render/ESP.kt

```diff
@@ -55,6 +57,22 @@ object ESP : Module("ESP", ModuleCategory.RENDER) {
         private val colorGreen by IntegerValue("G", 255, 0..255) { !colorRainbow }
         private val colorBlue by IntegerValue("B", 255, 0..255) { !colorRainbow }
 
+    private val maxRenderDistance by object : IntegerValue("MaxRenderDistance", 100, 1..200) {
+        // TODO: find better way?
+        override fun onInit(value: Int) {
+            maxRenderDistanceSq = value.toDouble().pow(2.0)
+        }
+
+        override fun onUpdate(value: Int) {
+            maxRenderDistanceSq = value.toDouble().pow(2.0)
+        }
+    }
+
+    private val onLook by BoolValue("OnLook", false)
+    private val maxAngleDifference by FloatValue("MaxAngleDifference", 5.0f, 5.0f..90f) { onLook }
+
+    private var maxRenderDistanceSq = 0.0
```

Comment:
My suggestion is to change it to:
`private var maxRenderDistanceSq = 10000.0 `
and get rid of onInit(value) method in maxRenderDistance object.
Why 10000.0? Because it's (default value)^2
It will help if there is no config file 
and maxRenderDistanceSq will be changed for sure if there is a config file.

It can be done in the same way with other "squared" variables which are dependent on configurable options. 
By the way, it applies to ChestAura as well =) (The same issue)

Reply:
alright